resources/lang/zh-hant.json

```diff
@@ -0,0 +1,221 @@
+{
+	"meta": {
+		"language": {
+			"value": "zh-hant",
+			"updated": "1.5"
+		},
+		"native": {
+			"value": "中文 (繁體)",
+			"updated": "1.5"
+		},
+		"authors": {
+			"value": [
+				"PEMapModder",
+				"旋風之音 (GoneTone)"
+			],
+			"updated": "1.0"
+		},
+		"comment": {
+			"value": "主要的版本會在每一次新的插件釋放時會有衝突本次的版本應該會修正此提交(包括合併提交)的en.json文件會被更新。所有未使用的翻譯作品仍必須有`更新`的權力版本被定義在meta.language.updated，有關英文的版本則被定義在meta.native.updated",
```

Comment:
You actually don't need to translate this. These 4 lines should be deleted :P But whatever, it is OK.